cypress.config.js

```diff
@@ -0,0 +1,41 @@
+import { defineConfig } from 'cypress';
+import plugins from './cypress/plugins/index.js';
+
+export default defineConfig({
+  viewportWidth: 1200,
+  viewportHeight: 900,
+  video: false,
+  numTestsKeptInMemory: 20,
+  defaultCommandTimeout: 10000,
+  retries: {
+    runMode: 2,
+    openMode: 0
+  },
+  env: {
+    '@apostrophecms-pro/cypress-tools': {
+      assembly: true,
+      apiKey: 'cypressAPIKey',
```

Comment:
Does this allow random people on the network to talk to your Cypress instance if they know this hardcoded string?

Reply:
Yes. It's how API keys work and Cypress need those in order to allow related commands to manage data. Going for yet another environment over complicates the whole.

Reply:
Also the key is configured only when your run the app in `CI=1` mode.